src/tree.c

```diff
@@ -369,12 +369,8 @@ struct policy_node *dfs_next(const struct policy_node *node)
 	}
 }
 
-enum selint_error free_policy_node(struct policy_node *to_free)
+static void free_single_policy_node(struct policy_node *to_free)
```

Comment:
Maybe "free_single_policy_node_data" would be a better name here?  Since it doesn't free the rest of the node.